backend/apps/onboarding/tasks.py

```
@@ -0,0 +1,111 @@
+    total = journeys.count()
+    if total > 0:
+        metrics.discovery_dropoff = (metrics.discovery_count / total) * 100
+        metrics.setup_dropoff = (metrics.setup_count / total) * 100
+        metrics.first_issue_dropoff = (metrics.first_issue_count / total) * 100
+        metrics.pr_dropoff = (metrics.pr_submitted_count / total) * 100
+
+    # Average durations
+    completed = journeys.filter(status='completed')
+    if completed.exists():
+        metrics.avg_discovery_duration = completed.aggregate(
+            avg=models.Avg('discovery_duration')
```

Ruff (0.15.21)

[error] Undefined name `models`

(F821)